java/pending_code_changes.txt

```diff
@@ -1 +1,5 @@
-
+Code changes:
+ - Using new possibleLengthInfo to decide whether a short number is the right
+length or not. This could result in more specific results; whereas before, a
```

Comment:
What impact does this have on the versioning scheme? Could argue for x.0.0 or x.y.z - what did we do for regular numbers?